test/init.go

```diff
@@ -62,3 +67,32 @@ func Initialize() {
 	}
 	util.InitializeTemplates(jt, cjt)
 }
+
+// envInit sets up environmental variables for eunomia tests
+func envInit() {
+	logf.Log.Info("Setting up environmental variables")
+	_, initFilename, _, _ := runtime.Caller(0)
+	eunomiaRoot := filepath.Join(filepath.Dir(initFilename), "..")
+	envs := map[string]string{
+		"JOB_TEMPLATE":     filepath.Join(eunomiaRoot, "./build/job-templates/job.yaml"),
+		"CRONJOB_TEMPLATE": filepath.Join(eunomiaRoot, "./build/job-templates/cronjob.yaml"),
+		"WATCH_NAMESPACE":  "",
+		"OPERATOR_NAME":    "eunomia-operator",
+		"GO111MODULE":      "on",
```

Comment:
why is this needed? does it even have any effect, setting it when we're already inside `go test`?